conf/app.routes

```diff
@@ -10,6 +10,12 @@ PUT           /agent/:arn/service/HMRC-MTD-VAT/client/VRN/:vrn               @uk
 GET           /agent/:arn/service/HMRC-MTD-VAT/client/VRN/:vrn               @uk.gov.hmrc.agentclientrelationships.controllers.RelationshipsController.checkWithMtdVat(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn, vrn: uk.gov.hmrc.agentmtdidentifiers.model.Vrn)
 DELETE        /agent/:arn/service/HMRC-MTD-VAT/client/VRN/:vrn               @uk.gov.hmrc.agentclientrelationships.controllers.RelationshipsController.deleteVatRelationship(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn, vrn: uk.gov.hmrc.agentmtdidentifiers.model.Vrn)
 
+
+#NI-ORG relationship
+PUT           /agent/:arn/service/HMRC-NI-ORG/client/EORI/:eori             @uk.gov.hmrc.agentclientrelationships.controllers.RelationshipsController.createForNiOrg(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn, eori: uk.gov.hmrc.agentmtdidentifiers.model.Eori)
```

Comment:
this may be out of the scope of this PR, but all PUTs  are not doing anything service specific , so can we remove `service/HMRC-NI-ORG` part from the urls ?